scripts/bootstrap.py

```diff
@@ -104,6 +105,7 @@ def populate_known_hosts(hosts, dest_file):
 
 def main():
     try:
+        jenkins_agent_role = os.environ['JENKINS_AGENT_ROLE']
```

Comment:
Let's make this required and set the default in the logic for the Universe package (as we are doing for #94).